src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -166,6 +166,7 @@ public void SendAsync_ExpectedDiagnosticSourceNoLogging()
             }, UseSocketsHttpHandler.ToString()).Dispose();
         }
 
+        [ActiveIssue(27550)]        // NTLM source files seem to cause an event source conflict
```

Comment:
It's the NetEventSource.Security.cs file that got included.  The way we set this up is that there's a shared NetEventSource.Common.cs that every System.Net project includes.  It provides the bulk of the NetEventSource implementation.  But every project needs its own distinct EventSource from an ETW / EventSource perspective, which means every assembly needs its own distinct identity for the EventSource in that file.  So the class defined in NetEventSource.Common.cs is partial, and then each project includes its own, generally very small, NetEventSource.ProjectName.cs, which includes another part of that partial class that adds the appropriate attributes specific to that project, e.g. Name, Guid, etc.  The one included in System.Net.Http is NetEventSource.Http.cs.

If there's stuff in NetEventSource.Security.cs that needs to be shared across projects, we should move / refactor it into the .Common.cs.  We really need to avoid breaking the tracing in this lib... it's too valuable in finding issues.

Reply:
Yeah, the problem is that NetEventSource.Security.cs defines events that conflict (same id) with NetEventSource.Http.cs.

I'll look at refactoring this.

Reply:
I moved the event id definitions from NetEventSource.Security.cs to NetEventSource.cs so they no longer conflict.